test/test_piece_picker.cpp

```diff
@@ -225,7 +225,7 @@ bool verify_pick(std::shared_ptr<piece_picker> p
 	// make sure there are no duplicated
 	std::set<piece_block> blocks;
 	std::copy(picked.begin(), picked.end()
-		, std::insert_iterator<std::set<piece_block> >(blocks, blocks.end()));
+		, std::insert_iterator<std::set<piece_block>>(blocks, blocks.end()));
```

Comment:
hm. it seems like the iterator here could just be constructed with: std::back_inserter(blocks)